js/functions.js

```diff
@@ -297,11 +314,9 @@ function generateStatistics( data ) {
     }
     return HTML;
 }
-for ( var i = 0; i < statisticsInfo.length; i++){
-    statisticsInfo[i].score;
-}
-var zero = 0;
 
+        //effect of numbers counting
+var zero = 0;
```

Comment:
Kintamamojo `zero` pavadinimas be konteksto labai neaiškus, galbūt pabandyčiau sugalvoti aiškesnį variantą. Nes vėliau skaitant ir pati nesuprasi ką jis turi daryti.